tools/create-element/templates/element/element.ts

```diff
@@ -1,18 +1,14 @@
 import { LitElement, html } from 'lit';
 import { customElement } from 'lit/decorators.js';
 
-import { pfelement } from '@patternfly/pfe-core/decorators.js';
-
 import styles from '<%= cssRelativePath %>';
 
 /**
  * <%= readmeName %>
  * @slot - Place element content here
  */
-@customElement('<%= tagName %>') @pfelement()
+@customElement('<%= tagName %>')
 export class <%= className %> extends LitElement {
-  static readonly version = '{{version}}';
```

Comment:
I know we are moving toward removing individual published versions for the elements,  so this will would push us in that direction 👍